_pkgdown.yml

```diff
@@ -1,61 +1,48 @@
+url: epiforecasts.io/scoringutils/
```

Comment:
```suggestion
url: https://epiforecasts.io/scoringutils/
```

This needs to be a valid URL otherwise the sitemap won't be correct.

Reply:
thanks!